### ISSUE TITLE
For language rust annotation are not working

### ISSUE THREAD

**Issue by Proful** [code-hike/v1#14]
For rust mark annotation not working, if language changed to typescript it is working.
```rs
// !mark(1:2)
fn main() {
    let s1 = String::from("hello");
    let s2 = s1;

    println!("{}", s1);
    println!("{}", s2);
}
```

**Comment by pomber** [comment]
Should be fixed in v1.0.0-beta.2

**Comment by Proful** [comment]
Verified, working fine. Thanks!
